### PR TITLE
Fixed start UI camera picker using iOS integrated one (iPhone X)

### PR DESCRIPTION
**Pull request by mikeger**
### Issues

Users noticed that on iPhone X the custom picture picker camera UI is pretty broken. In other places of the app the custom UI seems to work fine.

### Solutions

I decided to replace the custom UI with iOS integrated one, since then we don't have the overhead of supporting it.

### CODE REVIEW

File: Wire-iOS/Sources/UserInterface/ConversationList/ConversationListViewController+StartUI.m [hunk 1]
```diff
@@ -26,12 +26,50 @@
...
+            
+            ZMConversation __block *conversation = nil;
+            [[ZMUserSession sharedSession] enqueueChanges:^{
+                Team *team = ZMUser.selfUser.team;
+
+                conversation = [ZMConversation insertGroupConversationIntoUserSession:[ZMUserSession sharedSession]
+                                                                     withParticipants:users.allObjects
+                                                                               inTeam:team];
+            } completionHandler:^{
+                dispatch_after(dispatch_time(DISPATCH_TIME_NOW, (int64_t)(0.3 * NSEC_PER_SEC)), dispatch_get_main_queue(), ^{
```
Comment (typfel) [inline Wire-iOS/Sources/UserInterface/ConversationList/ConversationListViewController+StartUI.m]: What purpose does this `dispatch_after` have?

Comment (mikeger) [reply Wire-iOS/Sources/UserInterface/ConversationList/ConversationListViewController+StartUI.m]: Moved from the old code: https://github.com/wireapp/wire-ios/pull/1543/files#diff-13928cc74df935b3538fe3d30deb4fafL238

Comment (mikeger) [reply Wire-iOS/Sources/UserInterface/ConversationList/ConversationListViewController+StartUI.m]: Will try to remove it, let's see if it's necessary.

**Comment by mikeger** [comment]
@typfel removed it, seems to be unnecessary.

**Comment by typfel** [comment]
Can't you delete `CameraViewController`? This was the last usage of it?

**Comment by mikeger** [comment]
@typfel I thought about it, but it is still used for picking user picture. There it works with iPhone X.

**Pull request merged**